service/history/service.go

```diff
@@ -62,6 +62,10 @@ type Config struct {
 	TransferProcessorUpdateAckInterval   time.Duration
 	TransferProcessorForceUpdateInterval time.Duration
 	TransferTaskWorkerCount              int
+
+	// Persistence settings
+	ExecutionMgrNumConns int
```

Comment:
any good reason these two should use a different connection pool ? They are both directing requests to the same keyspace, right ? Is the consistency level coupled with gocql.session today ?

Reply:
Agreed.  I just created two knobs since we have two separate clients for these.  We should clean it up when we can consolidate the gocql clients for all persistence interfaces.